java/Nikiforov-Oleksandr/src/com/zoolatech/lecture1/tasks/_9/Task9.java

```diff
@@ -0,0 +1,25 @@
+package com.zoolatech.lecture1.tasks._9;
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts an integer value and outputs all its
+ * digits in the reverse order on a new line.
+ */
+
+public class Task9 {
+    public static void main(String[] args) {
+        // task solution code here
+        Scanner sc = new Scanner(System.in);
+        reversedDigits(sc.nextInt());
+    }
+
+    public static void reversedDigits(int number){
+        //output all digits of the number in the reverse order
+        while(number != 0){
+            int digit = number%10;
+            System.out.println(digit);
+            number = (number - digit)/10;
```

Comment:
Why do we subtract the digit? The division operation will still remove it from the number.